include/zephyr/device.h

```diff
@@ -91,10 +91,8 @@ typedef int16_t device_handle_t;
  */
 #define DEVICE_NAME_GET(dev_id) _CONCAT(__device_, dev_id)
 
-/* Node paths can exceed the maximum size supported by
- * device_get_binding() in user mode; this macro synthesizes a unique
- * dev_id from a devicetree node while staying within this maximum
- * size.
+/* This macro synthesizes a unique dev_id from a devicetree node by using
+ * the node's dependency ordinal.
```

Comment:
I think you're modifying a comment for `Z_DEVICE_DT_DEV_ID()` and not for `DEVICE_NAME_GET()` as the commit description seems to suggest?

Reply:
Thanks for spotting, will fix the commit description along with the reviews :+1: